scripts/west_commands/runners/openocd.py

```diff
@@ -204,7 +206,7 @@ def supports_thread_info(self):
 
     def do_run(self, command, **kwargs):
         self.require(self.openocd_cmd[0])
-        if ELFFile is None:
+        if globals().get('ELFFile') is None:
```

Comment:
this whole check can be removed, it’s a hack we should not have merged in a first place

Reply:
> I think west could do something better, though.

That would indeed be nice and get a +1 from me, but would be for another day 🙂